test/integration/suites/nested-rotation/Dockerfile

```
@@ -0,0 +1,4 @@
+FROM spire-agent:latest-local AS openssl-agent-mashup
```

nit: maybe something more generic?
```suggestion
FROM spire-agent:latest-local AS nested-agent
```

---

done